tests/testthat/test-delineate.R

```diff
@@ -48,3 +48,26 @@ test_that("Delineate does not return the valley if the buffer method is used", {
                          suppressWarnings())
   expect_equal(names(delineations), "corridor")
 })
+
+test_that("If `network_buffer` is not specified, the default value is used", {
+  expect_message(with_mocked_bindings(get_osmdata = \(...) bucharest_osm,
+                                      get_dem = \(...) bucharest_dem,
+                                      delineate(city_name = "Bucharest",
+                                                river_name = "Dâmbovița",
+                                                crs = 32635) |>
+                                        suppressWarnings()),
+                 paste0("The default `network_buffer` of 3000 m ",
+                        "is used for corridor delineation."))
+})
+
+test_that("If `buildings_buffer` is not specified, the default value is used", {
+  expect_message(with_mocked_bindings(get_osmdata = \(...) bucharest_osm,
+                                      get_dem = \(...) bucharest_dem,
+                                      delineate(city_name = "Bucharest",
+                                                river_name = "Dâmbovița",
+                                                crs = 32635,
+                                                riverspace = TRUE) |>
+                                        suppressWarnings()),
+                 paste0("The default `buildings_buffer` of 100 m ",
+                        "is used for riverspace delineation."))
+})
```

Comment:
I think we previously removed similar tests to avoid tests on CRAN servers to take too much time. If we refactor delineate to move the data retrieval functions out of delineate as in #262, these tests wont be needed anymore, so let's maybe leave these out?

Reply:
@fnattino shouldn't then the previous two test be removed as well? If yes, we can do that with a separate PR focusing on refactoring `delineate()`

Reply:
Yes, you are right, let's do it in a separate PR, when we refactor delineate. So let's keep these here, forget my previous comment 🙂